.github/workflows/main.yml

```diff
@@ -7,8 +7,7 @@ jobs:
     strategy:
       matrix:
         env:
-          - {ROS_DISTRO: melodic, ROS_REPO: main, UPSTREAM_WORKSPACE: 'github:nobleo/tracking_pid#master'}
-          - {ROS_DISTRO: noetic, ROS_REPO: main, UPSTREAM_WORKSPACE: 'github:nobleo/tracking_pid#master'}
+          - {ROS_DISTRO: galactic, ROS_REPO: main}
```

Comment:
is ROS_REPO is the desired branch? and if that is the case, should we use galactic instead for the time being?

Reply:
It's the used branch for the rosdistro repository, not this repository